perfkitbenchmarker/providers/aws/aws_virtual_machine.py

```diff
@@ -138,9 +152,10 @@ def GetBlockDeviceMap(machine_type, root_volume_size_gb=None,
 
   if machine_type in NUM_LOCAL_VOLUMES:
     for i in xrange(NUM_LOCAL_VOLUMES[machine_type]):
-      mappings.append({
-          'VirtualName': 'ephemeral%s' % i,
-          'DeviceName': '/dev/xvd%s' % chr(ord(DRIVE_START_LETTER) + i)})
+      od = OrderedDict()
+      od['VirtualName'] = 'ephemeral%s' % i
```

Comment:
Unfortunately, that loses the ordering.  From https://docs.python.org/2.7/library/collections.html#collections.OrderedDict

> The OrderedDict constructor and update() method both accept keyword arguments, but their order is lost because Python’s function call semantics pass-in keyword arguments using a regular unordered dictionary.